src/inc/corinfo.h

```diff
@@ -2068,7 +2068,7 @@ struct CORINFO_Object
 struct CORINFO_String : public CORINFO_Object
 {
     unsigned                stringLen;
-    const wchar_t           chars[1];       // actually of variable size
+    wchar_t                 chars[1];       // actually of variable size
```

Comment:
No this is on Linux. I do not have the error with me, but it says that we are declaring a field const in a struct that has no constructor, meaning that it can never be initialized. This is causing a warning but in LLILC warnings are treated as errors, so it causes the build to fail.